samples/template-operator/charts/template-operator/Chart.yaml

```diff
@@ -3,6 +3,6 @@ apiVersion: v2
 name: template-operator
 description: A Helm chart for the Operator in a Cluster based on a Kustomize Manifest
 type: application
-version: v0.0.0
-appVersion: 0.0.0
+version: v0.0.4
```

Comment:
Why that particular version?

Reply:
because I tested and incremented a few, it is arbitrary for the sample

Reply:
👍